<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesTableForm</class>
 <widget class="QWidget" name="NotesTableForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>676</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="1">
    <widget class="QWidget" name="directionBtnsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QToolButton" name="rightBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/arrowRight.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="3">
       <widget class="QToolButton" name="blueBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/blueCube.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="leftBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/arrowLeft.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QToolButton" name="redBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/redCube.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="upBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/arrowUp.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="downLeftBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/arrowDownLeft.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="upRightBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/arrowUpRight.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="downRightBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/arrowDownRight.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="downBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/arrowDown.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="upLeftBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/arrowUpLeft.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="centerBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/arrowNone.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="infoWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="5">
       <widget class="QLineEdit" name="upDownCutsTF"/>
      </item>
      <item row="4" column="5">
       <widget class="QLineEdit" name="leftRightCutsTF"/>
      </item>
      <item row="6" column="2" colspan="4">
       <widget class="QWidget" name="infoBtnsWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="regeneratePB">
           <property name="text">
            <string>Regenerate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fillGapsPB">
           <property name="text">
            <string>Fill Large Gaps</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLineEdit" name="blueCutsTF"/>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="diagonalCutsL">
        <property name="text">
         <string>Diagonal:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="numLargsGapsTF">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="notesPerSecondTF">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="largestGapL">
        <property name="text">
         <string>Largest Gap</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="redCutsL">
        <property name="text">
         <string>Red Cuts:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="redCutsTF"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="notesPerSecondL">
        <property name="text">
         <string>Notes / Second</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="blueCutsL">
        <property name="text">
         <string>Blue Cuts:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="upDownCutsL">
        <property name="text">
         <string>Up/Down:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="numLargeGapsL">
        <property name="text">
         <string># Gaps &gt; 5 seconds</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="totalNotesTF">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="largestGapTF">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="difficultyL">
        <property name="text">
         <string>Easy</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="totalNotesL">
        <property name="text">
         <string>Total Notes</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="leftRightCutsL">
        <property name="text">
         <string>Left/Right:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLineEdit" name="diagonalCutsTF"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="playerControlsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="playRewindPB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/first.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="playPlayPB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/play.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="playPausePB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/pause.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="playLastPB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/Images/Images/last.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="viewBtnsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="viewEasyPB">
        <property name="text">
         <string>View Easy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewNormalPB">
        <property name="text">
         <string>View Normal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewHardPB">
        <property name="text">
         <string>View Hard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewExpertPB">
        <property name="text">
         <string>View Expert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewExpertPlusPB">
        <property name="text">
         <string>View Expert+</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="innerTableWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
